src/components/SearchView.tsx

```diff
@@ -86,25 +87,40 @@ export class SearchView extends React.Component<SearchViewProps, SearchViewState
     }
 
     onTextChange(event: any) {
+        clearTimeout(this.timeout);
```

Comment:
Just I checked the api doc of `setTimeout` on MDN, I found that in their example, they didn't assign timeout with any value when declaring or clearing it. Api link is here https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setTimeout